### PR TITLE
Add governance.md to WG Naming

### PR DESCRIPTION
**Pull request by zacharysarah**
This PR adds a governance document to WG Naming, described in #4968.

/wg naming
/assign @celestehorgan @jdumars @justaugustus

### CODE REVIEW

File: wg-naming/governance.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+- [Brigading](https://en.wikipedia.org/wiki/Vote_brigading)
+
+  Attempting to overrepresent a viewpoint through participation bias
+
+## Deviations from [sig-governance]
+
+WG Naming leads may close or end meetings at their discretion to shut down trolling or protect the safety of community members.
+
+WG Naming [restricts the scope of discussion][Scope of discussion].
```
Comment (nikhita) [inline wg-naming/governance.md:83]:
```suggestion
WG Naming [restricts the scope of discussion](#scope-of-discussion).
```

File: wg-naming/governance.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+  Attempting to overrepresent a viewpoint through participation bias
+
+## Deviations from [sig-governance]
+
+WG Naming leads may close or end meetings at their discretion to shut down trolling or protect the safety of community members.
+
+WG Naming [restricts the scope of discussion][Scope of discussion]. 
+
+[sig-governance]: "https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md"
```
Comment (nikhita) [inline wg-naming/governance.md:85]:
```suggestion
[sig-governance]: ./committee-steering/governance/sig-governance.md
```

File: wg-naming/governance.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# WG Naming governance
+
+## Uphold project values
+
+WG Naming upholds the Kubernetes project's [core values](https://github.com/kubernetes/community/blob/master/values.md).
```
Comment (nikhita) [inline wg-naming/governance.md:5]:
```suggestion
WG Naming upholds the Kubernetes project's [core values](./values.md).
```

File: wg-naming/governance.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  - A track record of supporting inclusion in the Kubernetes project
+
+WG Naming has many stakeholders, with many people eager to add their voices. WG Naming invites folks who are new to antiracism or inclusive language, and folks less experienced with frameworks of discrimination to participate by observing and learning. WG Naming invites contributors who meet prerequisites to contribute within the scope of WG Naming’s goals.
+
+### Resources for antidiscrimination
+
+https://developers.google.com/style/inclusive-documentation
+https://www.antiracismresources.info/#online-resources
+How to Be An Antiracist, Ibrahim X. Kendi
+So You Want to Talk About Race, Ijeoma Oluo
```
Comment (nikhita) [inline wg-naming/governance.md:28]: Can we add bullet points to these? Markdown renders this as one single line.

File: wg-naming/governance.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+### Scope of discussion
+
+WG Naming actively moderates discussion for speech that affirms the humanity of underrepresented minorities and affirms their ongoing experience of discrimination. For example: specific suggestions for replacement terms are welcome. Disagreement over the presence and impact of racism is not welcome.
+
+## Follow the community code of conduct
+
+Like all WGs and SIGs, WG Naming expects contributors to follow the [community code of conduct](CCoC).
```
Comment (nikhita) [inline wg-naming/governance.md:36]:
```suggestion
Like all WGs and SIGs, WG Naming expects contributors to follow the [Kubernetes Code of Conduct (CoC)]. 
```

File: wg-naming/governance.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+WG Naming actively moderates discussion for speech that affirms the humanity of underrepresented minorities and affirms their ongoing experience of discrimination. For example: specific suggestions for replacement terms are welcome. Disagreement over the presence and impact of racism is not welcome.
+
+## Follow the community code of conduct
+
+Like all WGs and SIGs, WG Naming expects contributors to follow the [community code of conduct](CCoC). 
+
+We raise this section’s topics as potential blockers to the work of WG Naming, since experience has shown they are a common and detrimental response.
+
+From the CCoC:
```
Comment (nikhita) [inline wg-naming/governance.md:40]:
```suggestion
From the CoC:
```

File: wg-naming/governance.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+> Examples of unacceptable behavior by participants include:
+>
+>   - Personal attacks
+>   - Trolling or insulting/derogatory comments
+
+**Note:** the CCoC governs conduct, not intent. WG Naming leads may refer unacceptable conduct to the CCoC committee regardless of professed intent.
+
+[community code of conduct]: "https://github.com/kubernetes/community/blob/master/code-of-conduct.md"
```
Comment (nikhita) [inline wg-naming/governance.md:49]:
```suggestion
[Kubernetes Code of Conduct (CoC)]: ./code-of-conduct.md
```

File: wg-naming/governance.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+We raise this section’s topics as potential blockers to the work of WG Naming, since experience has shown they are a common and detrimental response.
+
+From the CCoC:
+
+> Examples of unacceptable behavior by participants include:
+>
+>   - Personal attacks
+>   - Trolling or insulting/derogatory comments
+
+**Note:** the CCoC governs conduct, not intent. WG Naming leads may refer unacceptable conduct to the CCoC committee regardless of professed intent.
```
Comment (nikhita) [inline wg-naming/governance.md:47]:
```suggestion
**Note:** the CoC governs conduct, not intent. WG Naming leads may refer unacceptable conduct to the Code of Conduct Committee regardless of professed intent.
```

File: wg-naming/governance.md (hunk 4 above)
Comment (celestehorgan) [reply wg-naming/governance.md:28]:
```suggestion
- https://developers.google.com/style/inclusive-documentation
- https://www.antiracismresources.info/#online-resources
- _How to Be An Antiracist_, Ibrahim X. Kendi
- _So You Want to Talk About Race_, Ijeoma Oluo
```

For laziness

Comment (nikhita) [reply wg-naming/governance.md:28]: Thanks :pray:

**Comment by zacharysarah** [comment]
@nikhita Thanks for the review! 🙇  Commits are squashed and ready.

**Comment by dims** [comment]
/approve

please wait for @nikhita

File: wg-naming/governance.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
+# WG Naming governance
+
+## Uphold project values
+
+WG Naming upholds the Kubernetes project's [core values](./values.md).
```
Comment (cblecker) [inline wg-naming/governance.md:5]:
```suggestion
WG Naming upholds the Kubernetes project's [core values](/values.md).
```

File: wg-naming/governance.md [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+From the CoC:
+
+> Examples of unacceptable behavior by participants include:
+>
+>   - Personal attacks
+>   - Trolling or insulting/derogatory comments
+
+**Note:** the CoC governs conduct, not intent. WG Naming leads may refer unacceptable conduct to the Code of Conduct Committee regardless of professed intent.
+
+[Kubernetes Code of Conduct (CoC)]: ./code-of-conduct.md
```
Comment (cblecker) [inline wg-naming/governance.md:49]:
```suggestion
[Kubernetes Code of Conduct (CoC)]: /code-of-conduct.md
```

File: wg-naming/governance.md [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+
+  Attempting to overrepresent a viewpoint through participation bias
+
+## Deviations from [sig-governance]
+
+WG Naming leads may close or end meetings at their discretion to shut down trolling or protect the safety of community members.
+
+WG Naming [restricts the scope of discussion](#scope-of-discussion).
+
+[sig-governance]: ./committee-steering/governance/sig-governance.md
```
Comment (cblecker) [inline wg-naming/governance.md:85]:
```suggestion
[sig-governance]: /committee-steering/governance/sig-governance.md
```

File: wg-naming/governance.md [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+
+## Uphold project values
+
+WG Naming upholds the Kubernetes project's [core values](./values.md).
+
+### Inclusive is better than exclusive
+
+> Broadly successful and useful technology requires different perspectives and skill sets which can only be heard in a welcoming and respectful environment.
+
+This document covers some specific ways that respect and welcome look like in WG Naming.
```
Comment (cblecker) [inline wg-naming/governance.md:11]: This sentence doesn't appear to be grammatically clear/correct

**Comment by nikhita** [comment]
Sorry, I messed up the links in the suggestions. :see_no_evil: 

Lgtm from me after @cblecker's comments are addressed. :+1:

File: wg-naming/governance.md (hunk 12 above)
Comment (zacharysarah) [reply wg-naming/governance.md:11]: It is correct, but I can make it clearer.

Comment (zacharysarah) [reply wg-naming/governance.md:11]: Also—I am on vacation for the next two weeks, starting yesterday. I will address this feedback, and ask that you consider making any further feedback iterable by another WG Naming lead in a subsequent PR.

**Comment by zacharysarah** [comment]
@cblecker Thanks for the feedback, addressed in https://github.com/kubernetes/community/commit/5c4f934fa2c316f73a71127e6e46df08f7799a97.

@nikhita 👋

**Comment by nikhita** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**